source/_integrations/neptune_apex.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Neptune Apex
+description: Instructions on how to integrate a Neptune Systems Apex aquarium controller within Home Assistant.
+ha_category:
+  - Hub
+  - Sensor
+  - Light
+ha_release: "0.110"
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@jpelzer'
+ha_domain: neptune_apex
+---
+
+The Neptune Apex integration provides connectivity to a Neptune Systems Apex Aqaucontroller.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Sensor](#sensor)
+- [Light](#light)
+
+## Configuration
+
+Home Assistant offers Neptune Apex integration through **Configuration** -> **Integrations** -> **Neptune Apex**. Follow the instructions to get it set up.
+
+{% configuration %}
+host:
+  description: The domain name or IP address of the Apex, e.g., 192.168.1.1.
+  required: true
+  type: string
+username:
+  description: The username required to connect to the Apex.
+  required: true
+  type: string
+password:
+  description: The password required to connect to the Apex.
+  required: true
+  type: string
+{% endconfiguration %}
```

Comment:
This configuration block is used for YAML configuration. Please remove it.

Reply:
Done.